pkg/security/tests/module_tester.go

```diff
@@ -872,10 +874,25 @@ func newTestModule(t testing.TB, macroDefs []*rules.MacroDefinition, ruleDefs []
 		return nil, err
 	}
 
-	if _, err = setTestPolicy(st.root, macroDefs, ruleDefs); err != nil {
+	if _, err = setTestPolicy(st.root, macroDefs, ruleDefs, fmt.Sprintf(testPolicyID, 0)); err != nil {
 		return nil, err
 	}
 
+	if opts.nbPoliciesToLoad > 1 {
```

Comment:
could we use the opts.preStartCallback options to move the extra policy loading code into the dedicated tests ?

Reply:
Done in https://github.com/DataDog/datadog-agent/pull/19583/commits/d6075ed533a5f6e27b1bf8da65414bd277b224b7 !